### ISSUE TITLE
Default shortcuts are too long.

### ISSUE THREAD

**Issue by MicahZoltu** [mintty/wsltty#59]
I wanted to change my shell so I attempted to follow the instructions in the readme.  I cracked open the `Ubuntu in Mintty` shortcut in `%APPDATA%\Microsoft\Windows\Start Menu\Programs\WSLtty` and attempted to edit the Target.  Unfortunately, it doesn't have `/bin/bash` in it and attempting to add `-t /bin/bash` fails because the Target text field is too full to add anything else.  It also appears that the line is truncated, though the shortcut seems to work none the less.

This is what the Target field has:
```
%LOCALAPPDATA%\wsltty\bin\mintty.exe --wsl --rootfs="C:\Users\micah\AppData\Local\Packages\CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc\LocalState/rootfs" -h err --configdir="%APPDATA%\wsltty" -o Locale=C -o Charset=UTF-8 /bin/wslbridge --distro-guid "
```

Another shortcut was truncated even sooner:
```
%LOCALAPPDATA%\wsltty\bin\mintty.exe --wsl --rootfs="C:\Users\micah\AppData\Local\Packages\CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc\LocalState/rootfs" -h err --configdir="%APPDATA%\wsltty" -o Locale=C -o Charset=UTF-8 /bin/wslbridge -C~ --distro-gu
```

----

On a related note, what is the right way to create a shortcut that uses a custom shell (fish in my case) and has a custom start directory?

**Comment by MicahZoltu** [comment]
To answer my own question at the end (though there is still a problem with docs and shortcut length):
`-C/mnt/c/path/to/default/dir` will change the default path and there are a couple `.bat` files in the wsltty install directory that have un-truncated commands.

**Comment by mintty** [comment]
The line is only truncated in the Explorer Properties menu. However, I am aware this is not satisfying. I can try to shorten or reduce the parameters a little bit...
But what can we do if MS forces us to refer to 2 different super-long cryptic IDs (rootfs, distro-guid)?

**Comment by mintty** [comment]
Released 1.8.0.
